agent/agent_worker.go

```diff
@@ -706,6 +706,16 @@ func (a *AgentWorker) Disconnect(ctx context.Context) error {
 	return nil
 }
 
+// This monitor has a 3rd implicit state we will call "initializing" that all agents start in
+// Agents can transition to busy and/or idle but always start in the "initializing" state
+/*
+//                - Busy
+//              /    |
+// Initializing      |
+//              \    |
+//                - Idle
```

Comment:
how about an ascii dag?

```suggestion
//                -> Busy
//              /     ^
// Initializing       |
//              \     v
//                -> Idle
```

Reply:
Honestly I couldn't be bothered figuring out how to do a down arrow 😆